dynawo/sources/Solvers/AlgebraicSolvers/DYNSolverKINAlgRestoration.cpp

```diff
@@ -45,9 +45,11 @@ using boost::shared_ptr;
 
 namespace DYN {
 
-SolverKINAlgRestoration::SolverKINAlgRestoration() :
+SolverKINAlgRestoration::SolverKINAlgRestoration() : SolverKINCommon(),
```

Comment:
Please respect the coding style and put this init on a new line (like mode_)

Reply:
OK